NXP-code-hub/FRDM-MCXN947/zephyr-hello-server/src/main.c

```diff
@@ -0,0 +1,299 @@
+/*
+ * Copyright (c) 2016 Intel Corporation
```

Comment:
Please fix licensing. This code looks like ours and it should be labeled as such.